src/Helper.java

```diff
@@ -0,0 +1,20 @@
+import java.util.Scanner;
+public class Helper {
+    private static final Scanner scanner = new Scanner(System.in);
+
+        public static String getInput(String prompt) {
+        System.out.print(prompt);
+
+        return null;
+    }
+
+    public static int getIntInput(String prompt) {
+        System.out.print(prompt);
+
+        return 0;
```

Comment:
Please Return prompt message (int value) from Scanner instead of 0.

Reply:
now return int value instead of 0. Kindly check.